cmd/publish.go

```diff
@@ -56,9 +56,13 @@ dapr publish --enable-domain-socket --publish-app-id myapp --pubsub target --top
 
 		client := standalone.NewClient()
 		// TODO(@daixiang0): add Windows support
-		if runtime.GOOS == "windows" && publishSocket != "" {
-			print.FailureStatusEvent(os.Stderr, "unix-domain-socket option still does not support Windows!")
-			os.Exit(1)
+		if publishSocket != "" {
+			if runtime.GOOS == "windows" {
+				print.FailureStatusEvent(os.Stderr, "unix-domain-socket option still does not support Windows!")
```

Comment:
I know that this message was here before, but since we are already making changes, can we consider something like `The unix-domain-socket option is not supported on Windows.`? The current message almost sounds like it is taking a jab at Windows which of course is unintentional.

Reply:
Yes! I pushed the changes.